dataactvalidator/scripts/read_zips.py

```diff
@@ -25,8 +24,44 @@
 chunk_size = 1024 * 10
 
 
-# update contents of state_congressional table based on zips we just inserted
+def hot_swap_zip_tables(sess):
+    """ Drop the existing zips table, rename the temp_zips table, and rename all the indexes in a transaction.
+
+        Args:
+            sess: the database connection
+    """
+    # Getting indexes before dropping the table
+    indexes = Zips.__table__.indexes
+
+    logger.info("Hot swapping temporary zips table to official zips table.")
+
+    # Do everything in a transaction so it doesn't affect anything until it's completely done
+    sess.execute("BEGIN;")
```

Comment:
Instead of multiple `execute` calls, I wonder if it'll be cleaner or faster to have everything but the index changes be as one string/file and one `execute`. That's my preference but this is optional.

Reply:
Agree.